components/atom/switch/src/index.scss

```diff
@@ -0,0 +1,121 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$h-atom-switch--m: $sz-icon-l !default;
+$w-atom-switch--m: 34px !default;
```

Comment:
are youtr sure all these values must be hardcoded ?
We have to base components on sui-theme variables as much as possible

Reply:
I asked the design guys, the numbers are wrong. Keep it with these numbers, I'll take care of it

Reply:
Hi! This has been updated to 40px, I'm not sure if we should use the `$sz-icon-xl` var or create new ones for component widths and heights. What do you think @davidbarna ?